client/src/components/breadcrumbs.jsx

```diff
@@ -0,0 +1,39 @@
+//import { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import './Breadcrumbs.css'
+
+// internal logic for breadcrumbs:
+// "/"   -->  ["home"]
+// "/one" --> ["home", "one"]
+// "/one/one_one" --> ["home", "one", "one_one"] --> home / one / one_one
+//
+// hint:
+// ["home", "one", "one_one"].join("/")
+
+const Breadcrumbs = ({ currentPath, fetchFiles }) => {
+  const pathnames = currentPath === '/' ? ['home'] : ['home', ...currentPath.split('/').slice(1)]
+
+  // const pathnames = currentPath === '/' ? [] : currentPath.split('/').slice(1) I don't understand we should show the wotld HOME or no)

```

Comment:
this should not be a comment in code. 
ask this question from your mentor or friend through other communication channels :)

Reply:
ok